scripts/mysql-files/upgrading/upgrade-3.2.0sql_Fix_Defaults.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ck_playertimes ALTER velStartXY SET DEFAULT '0', ALTER velStartXYZ SET DEFAULT '0', ALTER velStartZ SET DEFAULT '0',  ALTER velEndXY SET DEFAULT '0',  ALTER velEndXYZ SET DEFAULT '0',  ALTER velEndZ SET DEFAULT '0';
+ALTER TABLE ck_playerrank ALTER lastseen SET DEFAULT '-1';
```

Comment:
this file is not needed

Reply:
For people who have already upgraded to 3.2.0 , how will it get done for them?

Reply:
well to bad for them